dts/arm/st/l0/stm32l072.dtsi

```diff
@@ -19,6 +19,7 @@
 			interrupt-names = "usb";
 			num-bidir-endpoints = <8>;
 			ram-size = <1024>;
+			maximum-speed = "full-speed";
```

Comment:
This is not strictly required as this property is optional. 
When not available, USB controller set speed to the controller maximum available speed.
This is what is done in STM32 USB driver.
Anyway, this doesn't harm.